script.js

```diff
@@ -127,9 +127,22 @@ function renderQuestion() {
 }
 
 function endGame() {
-  console.log('game ended');
+  document.querySelector('.quiz-container').remove();
+  var d = document.getElementById("game-container")
```

Comment:
don't use ids and normal classes. You'll notice in my code whenever I do query select, I use the classname which begin with `js-` so if you want to query select something add a js- class in the html first and then use it. It's a best practice so that you know what things in html, your js is dependent on. Otherwise you might remove the class because you don't need it's css but you unknowingly broke the js